app/src/main/java/com/example/anfcodetest/MainAdapter.kt

```diff
@@ -9,27 +9,15 @@ import androidx.recyclerview.widget.RecyclerView
  * Created by Matt Kleman 09/04/20
  */
 
-class MainAdapter(val cardList: List<Card>): RecyclerView.Adapter<CustomViewHolder>() {
+class MainAdapter(private val cardList: List<Card>): RecyclerView.Adapter<CustomViewHolder>() {
```

Comment:
Since this value is only being accessed here, we can just declare it private! This also will help us with testing this class.